CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG for ASAB WebUI Components
 
+## 25.2.5
```

Comment:
@queuing4oranges To properly distribute the tag, you would need to change this in the package.json of the components as well.